src/jobs/release-by-version.yml

```diff
@@ -0,0 +1,45 @@
+description: |
+  Publish a new GitHub Release for a repository by finding the version file and CHANGELOG then publishing a
+  release if the found version does not match the previous version.
```

Comment:
Hmm. I find this a bit hard to follow, there's a lot of pieces here. Maybe: 
"Conditionally publish a new GitHub Release for a repository based on the version file. If the version in the version file does not match the latest Github Release, a new release will be generated. Release description will be parsed from CHANGELOG.md by default." ?

Reply:
sure